src/native/libs/System.Native/entrypoints.c

```diff
@@ -46,6 +47,7 @@ static const Entry s_sysNative[] =
     DllImportEntry(SystemNative_ReadStdin)
     DllImportEntry(SystemNative_GetSignalForBreak)
     DllImportEntry(SystemNative_SetSignalForBreak)
+#endif
```

Comment:
```suggestion
#endif // !defined(TARGET_APPLE) || defined(TARGET_OSX)
```

Reply:
I'd like to understand the relationship between `TARGET_APPLE` and `TARGET_OSX`. It would seem to me that OSX should be a subset of APPLE. If so, can't we just say `TARGET_OSX`?

Reply:
`!defined(TARGET_APPLE) || defined(TARGET_OSX)` is macOS **and Linux/Unix** but not Apple mobile platforms.

Reply:
We could add `TARGET_APPLE_MOBILE` (we have that one for MonoVM as well) if that would make things more readable.